### ISSUE TITLE
grouped track autoscaling

### ISSUE THREAD

**Issue by rbuels** [GMOD/jbrowse#254]
That's a good point.  A more powerful version of this idea would probably be to have a way to group tracks that should all autoscale together, based on them being the same type of data, or in the same group of tracks. 

On 05/13/2013 05:14 PM, Nathan Boley wrote:> Hi All,

> The new auto-scaling feature is very useful for viewing RNAseq data,
> but it also (ironically) makes it difficult to compare expression
> levels across samples. Would it be possible to add another scaling
> feature that scales to the same max over all samples , i.e.
> 'autoscale': 'page'? ( or, even better, over selected samples, but
> that seems like much more work and the former would be tremendously
> useful )
> 
> Thanks again for all of your hard work!
> 
> Best, Nathan

> 
> Gmod-ajax mailing list
> Gmod-ajax@lists.sourceforge.net
> https://lists.sourceforge.net/lists/listinfo/gmod-ajax

**Comment by lpryszcz** [comment]
I'm also interested in this feature!

**Comment by cmdcolin** [comment]
There was additional interest in this at the GCCBOSC JBrowse BOF session

The user described interest in setting things like the "max score" across multiple bigwig tracks, or at the very least, something easier than doing "Edit config" to edit the "max_score" param manually.

**Comment by vaneet-lotay** [comment]
@cmdcolin,  I would like to say I still have a lot of interest for this feature for the bigwig tracks that we use at Xenbase.  There definitely needs to be a feature which groups tracks together so that they scale the same way, perhaps by track category or better yet thru a custom list.

Please assign this if you can and keep it as a feature request for a possible future release of JBrowse, I would really appreciate it, and I think there's evidence here that others would too.

If there's already an applicable feature that can handle this function, please point me to it then, I had posted it on the mailing list as well.

Thanks,

Vaneet

**Comment by srobb1** [comment]
@cmdcolin @rbuels We definitely have interest in this feature for bigwig tracks.

Thanks,
Sofia

**Comment by vaneet-lotay** [comment]
@cmdcolin @rbuels , I just want to reaffirm that we still have a lot of interest in a new feature that will handle this type of scaling problem.....

Thanks,
Vaneet

**Comment by cmdcolin** [comment]
This feature is available in multibigwig if all your tracks are grouped into that track  https://github.com/elsiklab/multibigwig and can do global or local autoscaling on the group of tracks

I can see the appeal in doing it outside of multibigwig or in core, but I just thought I would mention it

**Comment by vaneet-lotay** [comment]
Yes this can work through multibigwig, @cmdcolin.  However there are limitations to how tracks are graphed in a multibigwig group compared to regular bigWig tracks....they both each have their pros and cons I guess for different purposes.

I guess I'm saying I still find 'grouped' autoscaling outside multibigwig appealing :)

**Comment by feedmewifi** [comment]
I want to add that I would really like to see this implemented!
